node/src/components/chainspec_loader.rs

```diff
@@ -223,11 +187,8 @@ impl ChainspecLoader {
             let chainspec_loader = ChainspecLoader {
                 chainspec,
                 root_dir,
-                reactor_exit: Some(ReactorExit::ProcessShouldExit(ExitCode::Abort)),
```

Comment:
(Linking #1966 for reference: A lot of the logic removed here will have to be reimplemented in #1966.)